### PR TITLE
Add colors.yaml file

### PR DESCRIPTION
**Pull request by tomlazar**
Adds  a colors.yaml file that controls the colors used for any colorize operation.

solves issue #73

### CODE REVIEW

**Comment by athityakumar** [comment]
@tomlazar - Sorry for not suggesting this beforehand, but does `--light` / `--dark` flags make sense? If `--light` flag is given, light_colors.yaml` can be used, else `dark_colors.yaml` can be used.

File: lib/colorls/core.rb [hunk 1]
```diff
@@ -233,29 +234,28 @@ def ls_line(chunk)
...
+        return [:folder, color, :folders] unless @all_folders.include?(key)
         key = @folder_aliases[key] unless @folder_keys.include?(key)
-        return [key, :blue, :folders]
+        return [key, color, :folders]
       end
 
-      key = content.downcase.to_sym
-
-      return [key, :green, :recognized_files] if @file_keys.include?(key)
+      color = @colors[:recongnized_file]
```
Comment (athityakumar) [inline lib/colorls/core.rb]: Fix typo in `:recongnized_file` to `: recognized_file`.

**Comment by tomlazar** [comment]
would the `dark` scheme be the current set of colors, and would `light` be a complete different set, or just `light_whatever`

**Comment by athityakumar** [comment]
@tomlazar - By `--light`, I mean a color scheme which looks good on light backgrounds. Similarly for `--dark`. Can you update colors

File: lib/colorls/core.rb [hunk 2]
```diff
@@ -268,28 +259,27 @@ def ls_line(chunk)
...
-      key = content.downcase.to_sym
-
-      return [key, :green, :recognized_files] if @file_keys.include?(key)
+      color = @colors[:recognized_file]
+      return [content.downcase.to_sym, color, :recognized_files] if @file_keys.include?(key)
 
       key = content.split('.').last.downcase.to_sym
-
-      return %i[file yellow unrecognized_files] unless @all_files.include?(key)
+      return [:file, @colors[:unrecognized_files], :unrecognized_files] unless @all_files.include?(key)
```
Comment (athityakumar) [inline lib/colorls/core.rb]: Fix typo `@colors[:unrecognized_files]` -> `@colors[:unrecognized_file]`

File: lib/yaml/dark_colors.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Main Colors
+unrecongnized_file: yellow
```
Comment (athityakumar) [inline lib/yaml/dark_colors.yaml]: Fix typo `unrecongnized_file:` -> `unrecognized_file:`

File: lib/yaml/light_colors.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Main Colors
+unrecongnized_file: light_yellow
```
Comment (athityakumar) [inline lib/yaml/light_colors.yaml]: Fix typo `unrecongnized_file:` -> `unrecognized_file:`

**Pull request merged**